bedmaker.py

```diff
@@ -352,7 +352,7 @@ def main():
         pm.run(cmd, temp)
         bedtype = get_bed_type(temp)
         if bedtype is not None:
-            cmd = f"bedToBigBed -type={bedtype} {temp} {chrom_sizes} {bigNarrowPeak}"
+            cmd = f"./bedToBigBed -type={bedtype} {temp} {chrom_sizes} {bigNarrowPeak}"
```

Comment:
maybe leave out the `./`  and add `bedToBigBed to PATH?  because we don't know if that always gonna be the working dir.